### PR TITLE
(DOCSP-40128): v1.7.3 release updates

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
Various updates for the 1.7.3 release.

## STAGING

- [Release Notes](https://preview-mongodbjeffallenmongo.gatsbyjs.io/cluster-sync/DOCSP-40128/release-notes/1.7/#1.7.3-release)
- [resume Behavior](https://preview-mongodbjeffallenmongo.gatsbyjs.io/cluster-sync/DOCSP-40128/reference/api/resume/#behavior)
- [Limitations](https://preview-mongodbjeffallenmongo.gatsbyjs.io/cluster-sync/DOCSP-40128/reference/limitations/#sharded-clusters)
- [Use mongosync on Sharded Clusters](https://preview-mongodbjeffallenmongo.gatsbyjs.io/cluster-sync/DOCSP-40128/multiple-mongosyncs/#use-mongosync-on-sharded-clusters)
  - Changed important admonition at top of page.
  - Removed "Performance" section. 
- [Connecting mongosync](https://preview-mongodbjeffallenmongo.gatsbyjs.io/cluster-sync/DOCSP-40128/connecting/)
  - All three sub-pages are updated. (Changes are single-sourced.)
- [mongosync - Read Preference](https://preview-mongodbjeffallenmongo.gatsbyjs.io/cluster-sync/DOCSP-40128/reference/mongosync/#read-preference)

## JIRA

https://jira.mongodb.org/browse/DOCSP-40128

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=665e2ca2cf5bec1c2913453a

### CODE REVIEW

File: source/multiple-mongosyncs.txt [hunk 1]
```diff
@@ -177,13 +178,6 @@ Use ``curl`` or another HTTP client to issue the :ref:`start
 The ``start`` command options must be the same for all of the ``mongosync``
 instances.
 
-.. _c2c-sharded-performance:
-
-Performance
------------
-
-.. include:: /includes/sharding-perf
```
Comment (edobranov) [inline source/multiple-mongosyncs.txt:185]: You may be able to remove this file too.

File: source/release-notes/1.7.txt [hunk 2]
```diff
@@ -15,6 +15,62 @@ Release Notes for mongosync 1.7
...
+  - The source cluster is MongoDB 6.0 or earlier.
+  
+  - The destination cluster is MongoDB 7.0 or later.
+  
+  - A capped collection's size was modified during a migration.
+
+- Fixed bug introduced in v1.0.0 where mongosync ignored the
+  ``changeStreamPreAndPostImages`` field in ``modify`` change events.
+  This could result in a destination collection having a different value
+  for this field compared to its source collection. mongosync now
```
Comment (edobranov) [inline source/release-notes/1.7.txt:52]:
```suggestion
  for this field compared to its source collection. Mongosync now
```
We're not super consistent with this on our end, but by itself, we tend to treat the word "mongosync" like a common noun unless we're explicitly code-stylizing it like `mongosync`.

I see that most other mentions in the public docs stylize it, so I'd be in favor of doing a broader update where we change mongosync/Mongosync to `mongosync`  for consistency's sake. It's probably better suited for a separate ticket if you agree -- I'm okay with whatever you decide here in the meantime.

**Review by edobranov — APPROVED** [review]
LGTM! Thanks for getting on this so quickly. Just had a couple suggestions, but feel free to handle/reject them however you'd like.

**Review by tfogo — APPROVED** [review]
LGTM!

File: source/release-notes/1.7.txt (hunk 2 above)
Comment (jeff-allen-mongo) [reply source/release-notes/1.7.txt:52]: Makes sense to me, thanks for pointing this out. I'll capitalize here per your suggestion and can create a follow-up ticket to standardize across the docs.

Comment (jeff-allen-mongo) [reply source/release-notes/1.7.txt:52]: Here is the ticket to handle the standardization: https://jira.mongodb.org/browse/DOCSP-40158

**Review by mmaville-mdb — COMMENTED** [review]
LGTM!

**Pull request merged**